docs/package.json

```diff
@@ -33,6 +33,7 @@
     "eslint-plugin-react": "^5.0.1",
     "highlight.js": "^9.0.0",
     "history": "^2.0.0",
+    "html-webpack-plugin": "^2.16.1",
```

Comment:
Ah, actually, that's my mistake. I need to update the webpack dev-server config to match the production one.